---
 cmake/dependencies.cmake | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/cmake/dependencies.cmake b/cmake/dependencies.cmake
index 96afe74..166783c 100644
--- a/cmake/dependencies.cmake
+++ b/cmake/dependencies.cmake
@@ -95,14 +95,6 @@ endif()
 
 # whereami library
 find_package(whereami QUIET)
-if(NOT TARGET whereami::whereami)
-    fetchcontent_declare(
-        whereami
-        GIT_REPOSITORY https://github.com/gpakosz/whereami
-        GIT_TAG e4b7ba1be0e9fd60728acbdd418bc7195cdd37e7 # master at 5/July/2021
-    )
-    fetchcontent_makeavailable(whereami)
-endif()
 
 # mio header-only lib (provides memory mapping for files)
 find_package(mio QUIET)
-- 
2.40.1

